app/screens/ProductDetailsScreen/ProductDetailsScreen.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Image, Text, SafeAreaView, ScrollView, Button, Alert, RefreshControl } from 'react-native';
+import { Header } from '../../components/Header/Header';
+import { IconBasket } from '../../components/icons/IconBasket';
+import { IconHeart } from '../../components/icons/IconHeart';
+import { IconArrowLeft } from '../../components/icons/IconArrowLeft';
+
+export const ProductDetailsScreen: React.FC = (): JSX.Element => {
+    const [refreshing, setRefreshing] = useState(false);
+    const product = {
+        attributes: {
+            price: '86.99',
+            description: 'Doloremque ab accusamus blanditiis et. Deleniti ma…oloribus ad. Non repellat maiores delectus rerum.',
+            name: 'Raw Edge T Shirt'
+        },
+        id: '16'
+    };
+
+    const wait = (timeout: number) => {
+        return new Promise(resolve => setTimeout(resolve, timeout));
+    }
+
+    const onRefresh = React.useCallback(() => {
+        setRefreshing(true);
+        wait(2000).then(() => setRefreshing(false));
+    }, []);
+
+    return (
+    <SafeAreaView>
+        <ScrollView
+            refreshControl={
+            <RefreshControl
+                refreshing={refreshing}
+                onRefresh={onRefresh}
+            />
+        }>
+            <Header>
+                <View>
+                    <IconArrowLeft fill={'#FFFFFF'} />
+                </View>
+                <View style={styles.headerIcons}>
+                    <IconHeart fill={'#FFFFFF'} />
+                    <View style={styles.headerIconRight}>
+                        <IconBasket fill={'#FFFFFF'} />
+                    </View>
+                </View>
+            </Header>
+            <View style={styles.container}>
+                <View style={styles.imgContainer}>
+                    <Image
+                        style={styles.img}
+                        source={{uri: `https://picsum.photos/id/${product.id}/250`}}
+                        resizeMode={'contain'}
+                    />
+                </View>
+                <View style={styles.cell}>
+                    <Text style={styles.cellValue}>{product.attributes.name}</Text>
+                    <Text style={styles.cellTitle}>{product.attributes.price}</Text>
+                </View>
+                <View style={styles.cell}>
+                    <Text style={styles.cellTitle}>Select color</Text>
+                    <View style={styles.btn}>
+                        <Button
```

Comment:
This
<img width="129" alt="Screenshot 2022-01-05 at 14 21 52" src="https://user-images.githubusercontent.com/58725560/148209792-69b4b508-71ac-4076-8bf0-02022e8a6a3e.png">
looks different from mockup
<img width="123" alt="Screenshot 2022-01-05 at 14 22 36" src="https://user-images.githubusercontent.com/58725560/148209840-838b60f4-5b38-4d6c-85bf-695aa240b77f.png">

the thing is - don't use Button component. It's rather rare case to use. Look at Pressable instead. And style it as you wish